src/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -62,8 +63,7 @@
   private final ScheduledExecutorService m_lobbyWatcherResetupThread = Executors.newScheduledThreadPool(1);
   private ServerGame m_iGame = null;
   private boolean m_shutDown = false;
-  @SuppressWarnings("deprecation")
-  private final String m_startDate = new Date().toGMTString();
+  private final String m_startDate = new SimpleDateFormat().format(new Date());
```

Comment:
This should be fairly easy to do, it seems that I can simply add a locale to the constructor of simpleDateFormat